client/state/imports/selectors/is-importer-locked.js

```diff
@@ -0,0 +1,17 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Returns a boolean indicating whether the current import is locked (true)
+ * or unlocked (false)
+ *
+ * @param {Object} state Global state tree
+ * @param {string} importerId Check the lock state of the importer with this id
+ * @return {boolean} Is the importer locked
+ */
+export default ( state, importerId ) =>
+	get( state, [ 'imports', 'importerLocks', importerId ], false );
```

Comment:
with a well-defined importer state atom then we can probably forego having so many individual and specific selectors. we can use `getImportState( siteId, importId )` or `getImportState( importId )` and then access the individual properties instead of having a separate selector for each property.

"well-defined" is key here, and it would be helpful to have docs somewhere showing what this state could look like.